index.js

```diff
@@ -99,24 +108,35 @@ function myMap(array, callback) {
 
 // ***** Callbacks - Question 1 *****
 
-// *** Uncomment the lines below to test
-// console.log("%cCallbacks - Question 1", "color: red")
+function triple (number){
+  return number *=3
+}
+
 
-// console.log(myMap([1,2,3,4,5], triple)) 
-// // => [3,6,9,12,15]
+//*** Uncomment the lines below to test
+console.log("%cCallbacks - Question 1", "color: red")
+
+console.log(myMap([1,2,3,4,5], triple)) 
+// => [3,6,9,12,15]
+
+console.log(myMap([2,4,6,8], triple))   
+// => [6,12,18,24]
 
-// console.log(myMap([2,4,6,8], triple))   
-// // => [6,12,18,24]
-// console.log("%c----------", "color: red") 
 
 
 // ***** Callbacks - Question 2 *****
 // *** Uncomment the lines below to test
-// console.log("%cCallbacks - Question 2", "color: red")
+console.log("%cCallbacks - Question 2", "color: red")
+
+function greet (name){
+  return (`Hi, ${name}`)
+}
```

Comment:
Excellent! Only small thing - there's no need for the parentheses in the return on line 132 here